src/item.rs

```diff
@@ -68,6 +68,16 @@ pub struct Item {
 
     #[serde(rename = "command")]
     pub kind: ItemKind,
+
+    pub mappers: Vec<Mapper>,
```

Comment:
I currently dont see any value in having multiple mappers per item. Actually, I can't even imagine Mappers besides Raw and Regex - which might be done with an Option rather than with an Enum.
see #34 as reference